app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -34,13 +37,38 @@
     private Navigator mNavigator;
     private FragmentManager mManager;
     private CommentFragment mFragment;
-
-    VideoDetailViewModel(Context context, FragmentManager manager, TimelineModel timelineModel) {
+    private UserModel mTimelineUser;
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
+
+                /**
+                 * @param item truyền vào khi người dùng click vào layout header item
+                 * check điều kiện nếu userModel từ profile gừi sang trùng với user người tạo
+                 * post thì không điều hướng sang activity profile mới
+                 */
+                @Override
+                public void onItemUserNameClick(TimelineModel item) {
+                    if (mTimelineUser != null && mTimelineUser.getId()
```

Comment:
ERROR: 'if' have incorrect indentation level 20, expected level should be one of the following: 12, 16.